doc/tutorial/basic/index.md

```diff
@@ -334,8 +334,12 @@ and the following to your `build.sbt`:
 jsEnv := new org.scalajs.jsenv.jsdomnodejs.JSDOMNodeJSEnv()
 {% endhighlight %}
 
-This will use the [`jsdom`](https://github.com/jsdom/jsdom) library to simulate a DOM in Node.js.
-Note that you need to install it separately using
+This will use the [`jsdom`](https://github.com/jsdom/jsdom) library to simulate a DOM in Node.js. You need to install it separately.
+Best is to install this locally for this project. Therefore, initialise a `package.json` using the default values:
+
+    $ npm init -y
```

Comment:
I believe `npm init private` would be more appropriate in this case.

Reply:
I am not familiar with that.  As long as it does not ask questions during a tutorial.